model/websocket_client.go

```diff
@@ -195,7 +192,7 @@ func (wsc *WebSocketClient) Listen() {
 			}
 
 			rawMsg = buf.Bytes()
-			event := WebSocketEventFromJson(bytes.NewReader(rawMsg))
+			event := WebSocketEventFromJson(&buf)
 			// Reset buffer.
 			buf.Reset()
```

Comment:
I am wondering if it's safe to use `rawMsg` (which points to the data in the buffer) after this call.

Reply:
Good catch. Unfortunately now it's going to make things ugly, because this is in a loop so I can't use defer. I _could_ use a function, but I am wondering about how readable it's going to make things.

Reply:
Can we reset once at the top of the loop?

Reply:
That's perfect :ok_hand: